microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -146,19 +196,32 @@ This will display the Store purchase UI to the user, where the user can view det
 
 In the `details` property of the payment response, a purchase token is returned.
 
+See also:
+
+MDN:
+* [PaymentRequest](https://developer.mozilla.org/docs/Web/API/PaymentRequest) at MDN > References > Web APIs > PaymentRequest > Payment Request API.
+   * [PaymentRequest: show() method](https://developer.mozilla.org/en-US/docs/Web/API/PaymentRequest/show) at MDN > References > Web APIs > PaymentRequest > Payment Request API.
+
+W3C:
+* [PaymentRequest interface](https://www.w3.org/TR/payment-request/#dom-paymentrequest) in _Payment Request API: Candidate Recommendation Draft_ at W3C.
+   * [show() method](https://www.w3.org/TR/payment-request/#dom-paymentrequest-show) in _Payment Request API: Candidate Recommendation Draft_ at W3C.
+
 
 <!-- ====================================================================== -->
 ## Acknowledging a purchase
 
 The payment response returns a _purchase token_ string, which can be used for direct communication between your server and the service provider beyond the Digital Goods API.  Such communication can allow you to independently verify information about the purchase before granting entitlements.
 
-Some stores might require that you (the developer) acknowledge a purchase after the purchase has succeeded, to confirm that the purchase has been recorded.
+Some stores might require that you (the developer) acknowledge<!-- todo: how? using which apis? --> a purchase after the purchase has succeeded, to confirm that the purchase has been recorded.
```

Comment:
how do you acknowledge?  what APIs to use?  link?
@runyuany 

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api?branch=pr-en-us-3324#acknowledging-a-purchase

Reply:
removed the "Acknowledging a purchase" section per offline discussion